src/main/java/ch/njol/skript/entity/EntityData.java

```diff
@@ -177,15 +177,28 @@ private final static class EntityDataInfo<T extends EntityData<?>> extends Synta
 		final String codeName;
 		final String[] codeNames;
 		final int defaultName;
+		@Nullable EntityType entityType;
```

Comment:
```suggestion
		final @Nullable EntityType entityType;
```
any reason for it not to be final?